4-4-2.py

```diff
@@ -0,0 +1,84 @@
+'''
+
+
+Вывести матрицу 2
+
+На вход программе подаются два натуральных числа nnn и mmm, каждое на отдельной строке — количество строк и столбцов в матрице. Далее вводятся сами элементы матрицы — слова, каждое на отдельной строке; подряд идут элементы сначала первой строки, затем второй, и т.д.
+
+Напишите программу, которая считывает элементы матрицы один за другим, выводит их в виде матрицы, выводит пустую строку, и снова ту же матрицу, но уже поменяв местами строки со столбцами: первая строка выводится как первый столбец, и так далее.
+
+Формат входных данных
+На вход программе подаются два числа nnn и mmm — количество строк и столбцов в матрице, далее идут n×mn \times mn×m слов, каждое на отдельной строке.
+
+Формат выходных данных
+Программа должна вывести считанную матрицу, за ней пустую строку, и ту же матрицу, но поменяв местами строки со столбцами. Элементы матрицы разделять одним пробелом.
+Тестовые данные 🟢
+
+Sample Input 1:
+
+4
+2
+и
+швец
+и
+жнец
+и
+на
+дуде
+игрец
+
+Sample Output 1:
+
+и швец
+и жнец
+и на
+дуде игрец
+
+и и и дуде
+швец жнец на игрец
+
+Sample Input 2:
+
+2
+3
+не
+в
+бровь
+а
+в
+глаз
+
+Sample Output 2:
+
+не в бровь
+а в глаз
+
+не а
+в в
+бровь глаз
+
+
+'''
+
+def print_matrix(row, col):
+    matrix = []
+    tmp = []
+    for r in range(row):
+        for c in range(col):
+            str_in = input().split()
+            tmp.append(str_in)
+        matrix.append(tmp)
+        tmp = []
+    for r in range(row):
+        for c in range(col):
+            print(str(*matrix[r][c]), end=' ')
+        print()
+    print()
+    for c in range(col):
+        for r in range(row):
+            print(str(*matrix[r][c]), end=' ')
```

Comment:
Do you really need the unpacking (`*`) here?

Reply:
I do so, according to the condition of the problem. So that the output is unsigned " ' "and without square brackets[ ].